### ISSUE TITLE
ignore_index=250

### ISSUE THREAD

**Issue by herleeyandi** [meetshah1995/pytorch-semseg#67]
Hello in the loss.py file I check it and find this line.
`loss = F.nll_loss(log_p, target, ignore_index=250,weight=weight, size_average=False)`
why you use 250?, and what does it means?

**Comment by zhoufengbuaa** [comment]
@herleeyandi I feel the same confusion. I just replaced it with 255. In my experiment, I set the background to 255. And 250, there is no pixel corresponding to the number.

**Comment by herleeyandi** [comment]
@zhoufengbuaa Yes me too. I am doing the same thing :smile:  .

**Comment by ChangliangSun** [comment]
Hi, @herleeyandi @zhoufengbuaa     I am training with Segnet. In my trainset, most of the pixel are background with labeled 0. So if I want to train a good model, should I set the loss like this:  `loss = F.nll_loss(log_p, target, ignore_index=0,weight=weight, size_average=True)`?
Thank you for your help.

**Comment by zhoufengbuaa** [comment]
@ChangliangSun ignore_index in the segmentation means that the loss on these pixels are not counting in the training procedure. So please consider you problem.
